src/styles/CitySelectionForm.css

```diff
@@ -0,0 +1,31 @@
+element {
+    border: none;
+    background-image: none;
+    background-color: transparent;
+    -webkit-box-shadow: none;
+    -moz-box-shadow: none;
+    box-shadow: none;
+}
+
+select {
+    width: 100%;
+    height: 40px;
+    margin: 20px auto;
+    border-radius: 5px;
+    text-align: center;
+}
+
+button {
+    width: 100%;
+    height: 40px;
+    /* margin-bottom: 25px; */
```

Comment:
Not sure if this is still being used, but if not then the commented code can be removed